master/custom/builders.py

```diff
@@ -202,6 +202,7 @@ def get_builders(settings):
         ("PPC64 AIX", "edelsohn-aix-ppc64", AIXBuild, UNSTABLE, NO_TIER),
         ("PPC64 AIX XLC", "edelsohn-aix-ppc64", AIXBuildWithXLC, UNSTABLE, NO_TIER),
         ("AMD64 FreeBSD", "ware-freebsd", UnixBuild, STABLE, TIER_3),
+        ("AMD64 FreeBSD", "opsec-fbsd14", UnixBuild, STABLE, TIER_3),
```

Comment:
I prefer to declare it as unstable for now:

```suggestion
        ("AMD64 FreeBSD", "opsec-fbsd14", UnixBuild, UNSTABLE, TIER_3),
```